### ISSUE TITLE
History-chart tooltip: Recent changes made navigating multiple signals captured at different timesteps harder

### ISSUE THREAD

**Issue by Oekn5w** [home-assistant/frontend#16203]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When having multiple signals that are sampled at different intervals, the values of which are similar, it has become very hard to move the mouse cursor to show the value of a specific curve, e.g for finding extremum values. See attached GIF of some heating temperature measurements.
It seems like predominantly the time of the cursor is taken into account and the y value determines mostly which curves are checked.

This is likely introduced by #15887 (going by comparing 20230110.0 to 20230404.1 and searching for "tooltip", although I don't have any experience in TS), the xy axis was actually mentioned in review. I guess the xy axis parameter should fix this, but I do not a dev environment set up to check.

### Describe the behavior you expected

- The tooltip shouldn't bounce up and down (to different ids) when moving mostly horizontal.
- I should be able to see the extremum by moving in proximity to it.

### Steps to reproduce the issue

1. Have multiple signals that are close in measurements, but sampled at different times (such signals might be difficult to build for reproduction)
2. Have them in a history-graph card
3. Try to navigate them

### What version of Home Assistant Core has the issue?

2023.4.4

### What was the last working version of Home Assistant Core?

2023.1.7

### In which browser are you experiencing the issue with?

Firefox 111.0.1 | Google Chrome 111.0.5563.147 

### Which operating system are you using to run this browser?

Windows 10 10.0.19045, also on Ubuntu 22.04

### State of relevant entities

```txt
N/A
```

### Problem-relevant frontend configuration

```yaml
type: history-graph
entities:
<list of 6 temperature entities>
refresh_interval: 1
hours_to_show: 16
```

### Javascript errors shown in your browser console/inspector

```txt
N/A
```

### Additional information

![ha-frontend-issue](https://user-images.githubusercontent.com/38046255/232331692-6241819f-1e6c-4298-b454-993619b61267.gif)

**Comment by ildar170975** [comment]
Confirm the issue.
Using the card became not possible in some cases - you just cannot select a point.
![hhh](https://user-images.githubusercontent.com/71872483/232355461-58082376-72d9-492c-9688-bd3181301fe4.gif)

**Comment by ildar170975** [comment]
Possible related to the same changes in HA: https://github.com/home-assistant/frontend/issues/16306

**Comment by DavidBSteele** [comment]
I've experienced the same issue recently. I think the problem is there seems to be a y-axis offset for the point being selected. So if you hold the cursor above the line you are interested in it tracks the values but not where the pointer is located. 

It makes the graph cards much less useful as even knowing the offset its extremely difficult to select the right point on the graph. I've found this happens on mobile and desktop browsers using Edge. 

Edge - Desktop Version 113.0.1774.35 (Official build) (64-bit) (Win 11 Version 10.0.22621 Build 22621)
         - Mobile Version 112.0.1722.59 (Android)
